tests/integration/test_orders_api.py

```diff
@@ -536,6 +536,222 @@ async def test_download_asset_md(tmpdir, session):
     assert Path(filename).name == 'metadata.json'
 
 
+@respx.mock
+@pytest.mark.asyncio
+async def test_checksum_success(tmpdir, order_description, oid, session):
+    # Mock an HTTP response for download
+    order_description['state'] = 'success'
+    dl_url1 = TEST_DOWNLOAD_URL + '/1?token=IAmAToken'
+    dl_url2 = TEST_DOWNLOAD_URL + '/2?token=IAmAnotherToken'
+    dl_url3 = TEST_DOWNLOAD_URL + '/2?token=IAmYetAnotherToken'
```

Comment:
The test isn't about tokens, so I find the endings of the URLs a bit misleading. How about this?

```suggestion
    dl_url1 = TEST_DOWNLOAD_URL + '/asset1'
    dl_url2 = TEST_DOWNLOAD_URL + '/assert2'
    dl_url3 = TEST_DOWNLOAD_URL + '/manifest'
```